main.go

```diff
@@ -304,11 +305,13 @@ func (c *Controller) updateVPAMetrics(vpa *autoscaling.VerticalPodAutoscaler) er
 	if vpa.Spec.ResourcePolicy != nil {
 		addAllowed := func(containerName, allowed, resource string, q resource.Quantity) {
 			labels := prometheus.Labels{
-				labelNamespace: vpa.ObjectMeta.Namespace,
-				labelName:      vpa.ObjectMeta.Name,
-				labelContainer: containerName,
-				labelAllowed:   allowed,
-				labelResource:  resource,
+				labelNamespace:     vpa.ObjectMeta.Namespace,
+				labelName:          vpa.ObjectMeta.Name,
+				labelContainer:     containerName,
+				labelAllowed:       allowed,
+				labelResource:      resource,
+				labelTargetRefName: vpa.Spec.TargetRef.Name,
```

Comment:
Should we have pod names? That would help us in the grafana pods dashboard directly.

Reply:
Same as below